distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,363 @@
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Organizations Tab (click *Add* to add an organization)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Organization Name
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|Primary contact number for this organization
+|
+|no
+
+|Email
+|Primary contact email for this organization
+|
+|no
+
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Contacts (click *Add* button to add contact info)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Contact Title
+|Contact Title
+|String
+|yes
+
+|Contact First Name
+|Contact First Name
+|String
+|yes
+
+|Contact Last Name
+|Contact Last Name
+|String
+|yes
+
+|Address
+|Address for listed contact
+|String
+|minimum one
+
+|Phone number
+|Contact phone number
+|
+|minimum one
+
+|Email
+|Contact email
+|String
+|minimum one
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Collections (Click *Add* to add Content Collection(s))
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Content Name
+|Name for this metadata content
+|string
+|yes
+
+|Content Description
+|Short description for this metadata content
+|string
+|no
+
+|Content Object Type
+|The kind of content object this will be. Default value should be used in most cases.
+|string
+|yes
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+===== Adding a Service Binding to a Node
+
+Advertise the methods other nodes use to connect to the local ${branding} instance.
+
+* Navigate to ${admin-console}.
+* Select ${ddf-registry} or ${ddf-catalog}.
+** (*Local Registry Nodes* tab is editable from either application.)
+* Click the name of the desired node.
+* Click the *Services* tab.
+* Click *Add* to add a service.
+* Expand new Service.
+* Enter Service name and details.
+* Click *Add* to add binding.
+* Select Service Binding type.
+** Select one of the defaults or _empty_ for a custom service binding.
+** If selecting _empty_, fill in all required fields.
+* Click Save.
+
+==== Publishing to Others
+
+Send details about the local ${branding} instance to other nodes.
+
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give node a unique local name.
+* Enter Registry Service (CSW) Url.
+* Confirm *Allow Push* is checked.
+* Click *Add* to save the changes.
```

Comment:
This step (Add) is the last step in the first method to publish to others. I would say the steps below are a second method, which requires a Remote Registry being added first.

The Third method would be what @brianfelix is currently working on. Publishing directly from the Node Information tab. Those steps should be added with his PR I would guess.